src/server/api.ts

```diff
@@ -155,7 +157,42 @@ export const createRouter = ({
    */
   router.get('/games/:name', async ctx => {
     const gameName = ctx.params.name;
-    const matchList = await db.listGames({ gameName });
+    const {
+      isGameover: isGameoverString,
+      updatedBefore: updatedBeforeString,
+      updatedAfter: updatedAfterString,
+    } = ctx.query;
```

Comment:
```suggestion
    } = ctx.request.body;
```

The other API endpoints use JSON body parameters instead of the query string, so we should be consistent here. This should also simplify the parameter parsing logic that follows.

Reply:
Is this even possible for get requests?

Reply:
Oh, right… sorry, I should have noticed that — of course everything else is using JSON body, but those are all POST requests, so the query string makes sense here! In that case, I think we’re all good. 👍